pkg/graphql/request_test.go

```diff
@@ -295,6 +295,35 @@ func TestRequest_OperationType(t *testing.T) {
 	})
 }
 
+func TestRequest_IsValidated(t *testing.T) {
+	t.Run("should return false when schema is not validated", func(t *testing.T) {
+		string := testSubscriptionDefinition
+		schema, _ := NewSchemaFromString(string)
+		request := Request{
+			OperationName: "Hello",
+			Variables:     nil,
+			Query:         `query Hello { hello }`,
+		}
+
+		result := request.IsValidated(schema)
+		assert.False(t, result)
+	})
+
+	t.Run("should return true when schema is successfully validated", func(t *testing.T) {
+		schema := starwarsSchema(t)
+		request := requestForQuery(t, starwars.FileHeroWithAliasesQuery)
+
+		result := request.IsValidated(schema)
+		assert.False(t, result)
+
+		valRes, _ := request.ValidateForSchema(schema)
+
+		result2 := request.IsValidated(schema)
+		assert.True(t, valRes.Valid)
+		assert.True(t, result2)
+	})
+}
```

Comment:
I think a test for `isValid` could make sense where we test for non-existent hashs